CMSgov/mint-app
# Easi 1638/data backfill

# EASI-1638 Data Backfill


Note this is a utility meant to be used once. It has already been used to import data, and likely won't be used again.

## Changes and Description

- Create an ETL Utility to transfer Sharepoint Data into MIN
- Add development level debug settings to vscode to allow VSCODE to get environment variables during testing / debugging locally.



## How to test this change
1. This requires a few git ignored files to work correctly, namely a data export csv from sharepoint, and a JSON of possible users.
2. Add those files (from box and an export from google sheets). 
3. Run the backfill command on your local db.
4. You can see the data uploaded into your local environment. You should also see the data exported and represented as a JSON in a variety of formats in the .gitignored folder


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
